styles.css

```diff
@@ -27,15 +27,25 @@ header {
   padding-right: 40px;
 }
 h3 {
-  padding-top: 5px;
+  padding-right: 50%;
+}
+#view-favs-button {
+  background-color: rgba(225, 225, 225, 1);
+  border-radius: 3px;
+  border: 0px;
+  font-size: 1vw;
+  margin-top: 15px;
+  margin-bottom: 15px;
+  padding-top: 7px;
+  padding-bottom: 7px;
 }
 #add-recipe-button {
   background-color: rgba(225, 225, 225, 1);
   border-radius: 3px;
   border: 0px;
-  font-size: 8pt;
```

Comment:
Changed fonts from pt to vw to allow for sizing changes based on viewing size of the window.